clash-lib/src/Clash/Core/Util.hs

```diff
@@ -618,9 +619,9 @@ shouldSplit0 tcm (TyConApp tcNm tyArgs)
     && nameOcc a2Nm == "Clash.Signal.Internal.KnownDomain"
   isHidden _ _ = False
 
-  -- Currently we're only interested in splitting of Clock, Reset, and Enable
```

Comment:
I removed this comment as it appeared stale to me: the list includes file handles as well. I don't think we need to update the comment, which things are split off by `splitTy` is pretty apparent from the code.